plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -231,7 +233,9 @@ protected void verifyConcurrentUpdateFailurePermissible(Exception e)
     protected void verifyConcurrentAddColumnFailurePermissible(Exception e)
     {
         assertThat(e)
-                .hasMessageContaining("Cannot update Iceberg table: supplied previous location does not match current location");
+                .hasMessageContaining("Failed to add column")
```

Comment:
Using `hasMessage` is recommended. Use "contains" weaker mathing only when message has varying part (but i prefer regex in such case), or is too long.
Having assertion on the whole message ensures the message is actually readable to the end user

Reply:
done